.github/workflows/ci.yml

```
@@ -26,35 +26,28 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - '1.3' # oldest supported version
-          - '1.6' # LTS
+          - min # oldest supported version
+          - lts # LTS
           - '1' # latest release
-          - 'nightly'
+          - pre # pre-releases
         os:
           - ubuntu-latest
           - macos-latest
           - windows-latest
         arch:
           - x64
     steps:
-      - uses: actions/checkout@v2
-      - uses: julia-actions/setup-julia@v1
+      - uses: actions/checkout@v4
+      - uses: julia-actions/setup-julia@v2.4.0
```

Any specific reason to include minor and patch part here?

---

In 2.4.0 support for the `min` version specifier was added. But the patch part is not needed, so probably we rather want
```suggestion
      - uses: julia-actions/setup-julia@v2.4
```